### ISSUE TITLE
False positive with ...ReferenceUsedNamesOnly.ReferenceViaFullyQualifiedName

### ISSUE THREAD

**Issue by simPod** [slevomat/coding-standard#1364]
```php
use function B\decode;

decode(\A\decode(''));
```

gives

> : Function \A\decode() should not be referenced via a fully qualified name, but via a use statement.

`B\decode` is already imported, `A\decode` should not be required to be imported then.

IIRC this works with classes

**Comment by kukulich** [comment]
Do you enabled `allowFullyQualifiedNameForCollidingFunctions` option ?

**Comment by simPod** [comment]
I'm using doctrine ruleset as a base and don't override this rule so yes 🤔 

https://github.com/doctrine/coding-standard/blob/f86c16aedb8a237f6c3deff01bd0131ac08d4570/lib/Doctrine/ruleset.xml#L309

**Comment by kukulich** [comment]
Fixed in https://github.com/slevomat/coding-standard/commit/9fa72f2a513a280f616f8a3a8b75ffb019bc7df2
